docs.rb

```diff
@@ -12,24 +12,17 @@ def self.plugins
 
 # gets readme files in the folders
 def self.get_readmes
+	c = File.open("docs.md", "w")
 	get_folders = plugins
-	# readmes = []
 	get_folders.each do |folder|
 		Dir.glob('./' + folder + '/*.md') do |md|
```

Comment:
yes 
I am trying to find a way of converting the md files to asciidoc , the plugins that am working with do not have .adoc files